load_pg_data.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
Would you kindly move this file somewhere inside, for example, `server/tests` or similar nested.

Reply:
Done 👍